install_config/certificate_customization.adoc

```diff
@@ -18,6 +18,24 @@ xref:../architecture/infrastructure_components/web_console.adoc#architecture-inf
 This can be done during a
 xref:../install/configuring_inventory_file.adoc#advanced-install-custom-certificates[cluster installation] or configured after installation.
 
+[[using-certificate-chains]]
+== Using Certificate Chains
+If a certificate chain is used, then all certificates must be manually
+concatenated into a single named certificate file. These certificates must
+be placed in the following order:
+
+* {product-title} master host certificate
+* Intermediate CA certificate
+* Root CA certificate
+* Third party certificate
+
+To create this certificate chain, concatenate the certificates into a
+common file, repeating as necessary for each certificate.
```

Comment:
I might break this in two sentences, maybe like  "... file. You must run this command for each certificate and ensure that they are in the correct order."

Reply:
I adjusted this to two sentences, but ended the second by referencing the defined order.